mne_bids/path.py

```diff
@@ -870,7 +867,35 @@ def find_empty_room(self):
             Returns None if none was found.
 
         """
-        return _get_matched_empty_room(self)
+        return _find_matched_empty_room(self)
+
+    @property
+    def empty_room(self):
+        """...
+        """
+        if self.datatype != 'meg':
+            raise ValueError('Empty-room data is only supported for MEG '
+                             'datasets') 
+
+        sidecar_fname = _find_matching_sidecar(self, extension='.json')
+        with open(sidecar_fname, 'r', encoding='utf-8') as f:
+            sidecar_json = json.load(f)
+
+        if 'AssociatedEmptyRoom' not in sidecar_json:
+            raise ValueError(
+                'The MEG sidecar file does not contain an '
+                '"AssociatedEmptyRoom" entry. Please use '
+                'BIDSPath.find_empty_room() to try and retrieve the '
```

Comment:
I would actually just update the existing method to use the json file to « find » the empty room. Find empty room would just look first in side car file and if not found look in the folders and if nothing is found crash.

Reply:
Done, we now do everything in `find_empty_room()`